<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chumstreamer</class>
 <widget class="QMainWindow" name="chumstreamer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChumStreamer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configureButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure...</string>
    </property>
   </widget>
   <widget class="QListWidget" name="musicFolderListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwdLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>empty text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playPauseButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>playpause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="songTitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="albumArtistLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="albumNameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="clickableList" name="artistListWidget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>271</width>
      <height>491</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="playlistListWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>256</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="repeatToggleButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>no repeat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextTrackButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;|</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomToggleButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>random off</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>clickableList</class>
   <extends>QWidget</extends>
   <header>clickablelist.h</header>
   <slots>
    <signal>Clicked()</signal>
    <signal>playlistAppend()</signal>
    <signal>playlistPrepend()</signal>
    <signal>ReturnKey()</signal>
    <signal>Back()</signal>
    <signal>Clear()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
